indico/core/config.py

```diff
@@ -328,7 +328,7 @@ class Config:
         'UseProxy'                  : False,
         'RouteOldUrls'              : False,
         'CustomCountries'           : {},
-        'PDFLatexProgram'           : 'pdflatex',
+        'PDFLatexProgram'           : 'xelatex',
```

Comment:
the comment in indico.conf.sample should be updated too

`
# Set the path to pdflatex if it's not within PATH:
# PDFLatexProgram = 'pdflatex'
```